float32[6] vel # [mrad/s]
float32[6] angle # [mrad]
uint32 msg_ctr # for detecting missed messages